themes/base.theme.bash

```diff
@@ -625,3 +625,14 @@ function _save-and-reload-history() {
 	local autosave=${1:-0}
 	[[ $autosave -eq 1 ]] && history -a && history -c && history -r
 }
+
+function conda_or_venv_prompt() {
+	local python_venv=""
+	if [[ -n "${CONDA_DEFAULT_ENV:-}" ]]; then
+		python_venv=$(condaenv_prompt)
+		PYTHON_VENV_CHAR=${CONDA_PYTHON_VENV_CHAR}
+	elif [[ -n "${VIRTUAL_ENV:-}" ]]; then
+		python_venv=$(virtualenv_prompt)
+	fi
+	[[ -n "${python_venv}" ]] && echo "${PYTHON_VENV_CHAR}${python_venv}"
+}
```

Comment:
This function should probably live closer to the [existing](https://github.com/Bash-it/bash-it/blob/3c2bc3d1967fe6e4cf3a065aa3a63eb8e8cf6ed2/themes/base.theme.bash#L478) functions.

NOTE: [`PYTHON_VENV_CHAR `](https://github.com/Bash-it/bash-it/search?q=PYTHON_VENV_CHAR) and [`CONDA_PYTHON_VENV_CHAR`](CONDA_PYTHON_VENV_CHAR) are only used by the themes that _do not_ invoke the `virtualenv_prompt` or `condaenv_prompt` functions.  Those function use `PYTHON_THEME_PROMPT_PREFIX` and `PYTHON_THEME_PROMPT_SUFFIX`

The spirit of the methods that live in this base theme is to use the `PREFIX` and `SUFFIX` variables.

I think this function just becomes:
```
	if [[ -n "${CONDA_DEFAULT_ENV:-}" ]]; then
		condaenv_prompt
	elif [[ -n "${VIRTUAL_ENV:-}" ]]; then
		virtualenv_prompt
	fi
```

Reply:
hi @petarnikolovski, want to continue and improve this?

Reply:
@NoahGorny Sorry for the late response, I will make moves on improvement soon, I couldn't manage because of my work schedule.